<launch>

    <node name="darp_path_plan_node" pkg="scale_map" type="darp_path_plan_node" output="screen">
    </node>
    <node name="scale_map_node" pkg="scale_map" type="scale_map_node" output="screen">
    </node>
    <node name="map_modify_node" pkg="scale_map" type="map_modify_node" output="screen">
    </node>
    <node name="line_detection_and_rotation_node" pkg="scale_map" type="line_detection_and_rotation_node" output="screen">
    </node>
</launch>
